api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillDescriptorTable.kt

```diff
@@ -29,12 +29,37 @@ object RichSkillDescriptorTable : LongIdTable("RichSkillDescriptor"), TableWithU
         onDelete = ReferenceOption.RESTRICT,
         onUpdate = ReferenceOption.CASCADE
     ).nullable()
-    val author = reference(
+}
+
+object RichSkillAuthors : Table("RichSkillAuthors") {
```

Comment:
I thought we agreed to extending the reuse of the RichSkillKeywords table, since Author is already implemented as a Keyword (Named Reference). Would you please help me understand the value in adding an extra table for this?

Reply:
`RickSkillKeywords` is how we have done it with all of the other named references. I see value in keeping that consistency. We can hide the `where` implementations with a few enums used as parameters. I would prefer that we not add a new table that is functionally redundant. If you see a compelling need, we can chat or speak in Slack? Please advise?